src/components/calculator.js

```diff
@@ -0,0 +1,33 @@
+function Calculator() {
+  return (
+    <div className="container">
+      <div className="calculator-grid">
+        <div className="display">
+          <div className="previou-operand">  </div>
+          <div className="current-operand"> 0</div>
+        </div>
+        <button type="button">AC</button>
+        <button type="button"> +/- </button>
+        <button type="button"> % </button>
+        <button type="button" className="operators"> + </button>
+        <button type="button"> 7 </button>
+        <button type="button"> 8 </button>
+        <button type="button"> 9 </button>
+        <button type="button" className="operators"> x </button>
+        <button type="button"> 4 </button>
+        <button type="button"> 5 </button>
+        <button type="button"> 6 </button>
+        <button type="button" className="operators"> - </button>
+        <button type="button"> 1 </button>
+        <button type="button"> 2 </button>
+        <button type="button"> 3 </button>
+        <button type="button" className="operators"> + </button>
+        <button type="button" className="span-two"> 0 </button>
+        <button type="button"> . </button>
+        <button type="button" className="operators"> = </button>
+      </div>
+    </div>
+  );
```

Comment:
- [ ] Dear @Cosmosha, As part of the main requirement for this project, your `Calculator.js` should have at least 1 child component. Please consider taking some logic into a separate component and including it in your `Calculator` component.

Reply:
Hello @TadesseDev, 
thank you for the feedback, however please can you simplify or help with me a clear direction on what I should do?
I am somehow confusion.

Reply:
Thank you so much @TadesseDev.